tkg/managementcomponents/helper.go

```diff
@@ -45,6 +45,18 @@ func GetTKGPackageConfigValuesFileFromUserConfig(managementPackageVersion string
 		return "", errors.Errorf("unknown provider type %q", providerType)
 	}
 
+	// get tkr-controller-config related values from user input
```

Comment:
It would be good if these could be grouped to a func.
Since this func is for configuring all tkgPackages, it might grow fast

Reply:
done